packages/nouns-webapp/src/pages/Vote/index.tsx

```diff
@@ -266,10 +268,10 @@ const VotePage = ({
         </div>
         {proposal && proposalActive && (
           <>
-            {showBlockRestriction && !hasVoted && (
+            {proposalCreationTimestamp && showBlockRestriction && !hasVoted && (
               <Alert variant="secondary" className={classes.blockRestrictionAlert}>
-                Only NOUN votes that were self delegated or delegated to another address before
-                block {proposal.createdBlock} are eligible for voting.
+                Only Nouns in your wallet or delegated to you before{' '}
```

Comment:
@lastpunk9999 I'm cool with this copy change but wonder if we should run it by the group for comment as well?

Reply:
"**Only Nouns in your wallet** or delegated to you before March 1, 2022 4:09 PM EST can vote." first part feels off.

How about "Only Nouns received or delegated to you before XXDATEXX can vote."?

Reply:
@ryeshrimp had a conversation offline about this with @cryptoseneca. Please change the wording to `Only Nouns you owned or were delegated to you before March 1, 2022 are eligible to vote.`